src/sentry/spans/buffer/redis.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from django.conf import settings
+from sentry_redis_tools.clients import RedisCluster, StrictRedis
+
+from sentry.utils import redis
+
+SEGMENT_TTL = 5 * 60  # 5 min TTL in seconds
+
+
+def get_redis_client() -> RedisCluster | StrictRedis:
+    return redis.redis_clusters.get(settings.SENTRY_SPAN_BUFFER_CLUSTER)
+
+
+def get_segment_key(project_id: int, segment_id: int) -> str:
+    return f"project:{project_id}:segment:{segment_id}"
```

Comment:
Perhaps consider giving a more specific key to indicate purpose. This is a very generic name and on other deployments (think single tenant/self hosted) where we use fewer redis instances, it could get confusing.